s4e-web/src/app/components/overlay-list/overlay-list.component.html

```diff
@@ -0,0 +1,241 @@
+<ng-container *ngIf="!(isLoading$ | async); else loadingRef">
+  <div *ngIf="!(overlays$ | async) || (overlays$ | async).length === 0" class="infobox infobox--emptylayers">
+    <p>
+      Nie masz jeszcze żadnej swojej nakładki. <a href="#">Sprawdź czym są nadkładki.</a>
```

Comment:
Where does this link lead?

Reply:
I don't know... I belive Andrzej would know